docs/source/xrpl.core.binarycodec.binary_wrappers.rst

```diff
@@ -0,0 +1,29 @@
+xrpl.core.binarycodec.binary\_wrappers package
+==============================================
+
+Submodules
```

Comment:
All of these docs files should be deleted - they're not needed, poetry just red-adds them every time an `__init__` file is edited

Reply:
I see thank you, I deleted them